vision/vision_general/scripts/gpsr_commands.py

```diff
@@ -0,0 +1,271 @@
+#!/usr/bin/env python3
+
+"""
+Node to handle GPSR commands.
+"""
+
+import cv2
+from ultralytics import YOLO
+import pathlib
+
+import rclpy
+from rclpy.node import Node
+from cv_bridge import CvBridge
+from sensor_msgs.msg import Image
+
+from frida_interfaces.srv import CountBy
+
+from ament_index_python.packages import get_package_share_directory
+
+package_share_dir = get_package_share_directory("vision_general")
+
+# Topics
+CAMERA_TOPIC = "/zed/zed_node/rgb/image_rect_color"
+
+COUNT_BY_COLOR_TOPIC = "/gpsr/count_by_color"
+COUNT_BY_CLOTHES_TOPIC = "/gpsr/count_by_clothes"
+COUNT_BY_PERSON_TOPIC = "/gpsr/count_by_person"
+COUNT_BY_OBJECTS_TOPIC = "/gpsr/count_by_objects"
+COUNT_BY_GESTURES_TOPIC = "/gpsr/count_by_gestures"
+COUNT_BY_POSE_TOPIC = "/gpsr/count_by_pose"
+IMAGE_TOPIC = "/gpsr/img_detection"
+
```

Comment:
@Ale-Coeto these constants should be located in frida constants I have seen some other prs where the case is the same for this related to vision 🥺